src/client/containers/BoardPage/MemberBoardPage.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import './BoardPage.style.css';
+import PropTypes from 'prop-types';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+import BoardImg from '../../assets/images/demo-boards-photos/Board1.jpg';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import { candidateListArr } from '../../components/CandidateCard/CandidateListArray';
+import { CardItemDecorator } from '../../components/CandidateCard/CandidateCardItem.component';
+import { candidateCardSorting } from '../../components/CandidateCard/CandidateCardSorting';
+import {
+  DragAndSortAdapter,
+  SortableItem,
+} from '../DragAndSortAdapter/DragAndSortAdapter';
+import { onDragEnd } from '../DragAndSortAdapter/OnDragEnd';
+
+export default function MemberBoardPage({ boardInfo }) {
+  console.log(boardInfo);
+  const [candidates, setCandidates] = useState(candidateListArr());
+  const deadlineDate = new Date(boardInfo.deadline);
+  // const deadlineDate = new Date('2021-09-12');
+  const today = new Date();
+
+  const onClick = () => {
+    // console.log('you clicked!');
+  };
+  console.log(boardInfo.title);
+  return (
+    <div className="Board-container">
```

Comment:
Similar duplication here. Please refactor. This also contains several subcomponents.
I see components for atleast:
Header
Title
Candidate List